src/main/java/edu/carrollcc/cis232/Question2.java

```diff
@@ -3,8 +3,35 @@
 public class Question2 {
 
 	public static void main(String[] args) {
-
+String t="Kayak";
+System.out.println(isPalindrome(t));
 	}
+	public static boolean isPalindrome(String s)  {
+		
+		return isPalindrome(s,s.indexOf(0)+1,s.length()-1);
 	
+		
+	}
+	public static boolean isPalindrome (String s,int index, int base )throws StringIndexOutOfBoundsException
+	{
+		//index=s.indexOf(0);
+		//base=s.lastIndexOf(s.length());
+		if(s.length()<0){
```

Comment:
The length will never be less than 0, you should check if the length is less than or equal to 1 for your base case. A string with 1 character or 0 characters is always a palindrome.